source/content/debug-cache.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Debug Caching Issues
+description: Learn how to identify and resolve caching issues affecting your Pantheon sites.
+categories: [performance]
+tags: [cache, cdn, cookies]
+contributors: [rachelwhitton]
+reviewed: "2020-10-14"
+---
+
+## Before You Begin
+First, verify caching configuration to ensure anonymous caching is enabled. Then test to determine if CDN caching is working on your site:
+
+- Enable anonymous caching within Drupal, [details](/drupal-cache)
+- WordPress sites on Pantheon have anonymous caching enabled by default, [details](/wordpress-cache-plugin)
+- [Testing Global CDN Caching](/test-global-cdn-caching)
+
+If you see `Age: 0` after multiple requests, your site is not caching properly.
+
+### Cache Related Headers
+<dl>
+ <dt>cache-control</dt>
+  <dd>Determines caching behaviors for the given request, this configuration is set by WordPress and Drupal. </dd>
+ <dt>age</dt>
+  <dd>How long the content has been stored in cache. If 0, the response was produced by WordPress or Drupal and not served from cache. </dd>
+ <dt>set-cookie</dt>
+  <dd>Used to send cookies from the application to the user agent. The platform will not cache a response that contains the `set-cookie` header.</dd>
```

Comment:
> Generated by WordPress or Drupal to send data back to the browser. The platform will not cache a response that contains the `set-cookie` header.

That ^ might be more helpful than my original: 
> Used to send cookies from the application to the user agent. The platform will not cache a response that contains the `set-cookie` header.

No strong preference, up to you all

Reply:
The markdown on `set-cookie` doesn't seem to work in the <dd> class

Reply:
I also have no preference, default to whichever you think is better.

Markdown was fixed with spacing, the MDX renderer needs an empty line to reset from HTML passthrough to Markdown interpretation.